launchcontainers/createsymlinks.py

```diff
@@ -123,17 +123,30 @@ def check_tractparam(lc_config, sub, ses, tractparam_df):
     container = lc_config["general"]["container"]
     precontainer_anat = lc_config["container_specific"][container]["precontainer_anat"]
     anat_analysis_num = lc_config["container_specific"][container]["anat_analysis_num"]
-    
-    fs_zip = os.path.join(
-        basedir,
-        "BIDS",
-        "derivatives",
-        precontainer_anat,
-        "analysis-" + anat_analysis_num,
-        "sub-" + sub,
-        "ses-" + ses,
-        "output", "fs.zip"
-    )
+    if run_lc:
+        rtp_analysis_num = lc_config["general"]["analysis_name"]
+        fs_zip = os.path.join(
+            basedir,
+            "BIDS",
+            "derivatives",
+            container,
+            "analysis-" + rtp_analysis_num,
+            "sub-" + sub,
+            "ses-" + ses,
+            "input",
+            "fs","fs.zip")
+    else:
+        fs_zip = os.path.join(
```

Comment:
if it is in this logic, then in the preparemode, you will read from the lc_config['container_specific']['anatrois'], and in run mode, you will read from lc_config['general']. this is contradict the logic of lauchcontainers, which in prepare mode, you prepare everything, and in run_mode, you only run the thing you have prepared.